packages/desktop-client/src/components/reports/GraphButton.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { forwardRef, type HTMLProps } from 'react';
+
+import { type CSSProperties, theme } from '../../style';
+import Button from '../common/Button';
+import HoverTarget from '../common/HoverTarget';
+import Text from '../common/Text';
+import { Tooltip } from '../tooltips';
+
+type GraphButtonProps = HTMLProps<HTMLButtonElement> & {
+  selected?: boolean;
+  style?: CSSProperties;
+  onSelect;
+  title?: string;
+  disabled?: boolean;
+};
+
+const GraphButton = forwardRef<HTMLButtonElement, GraphButtonProps>(
+  ({ selected, children, onSelect, title, style, disabled }) => {
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏doesn't seem like you're using the ref anywhere. So might as well remove `forwardRef`

```suggestion
const GraphButton = ({ selected, children, onSelect, title, style, disabled }: GraphButtonProps) => {
```